src/open_inwoner/conf/fixtures/django-admin-index.json

```
@@ -470,7 +470,7 @@
             "overige-diverse"
         ],
         "name": "Link naar Documentatie",
-        "link": "https://taiga.maykinmedia.nl/project/open-inwoner/wiki/home"
+        "link": "https://docs.openinwoner.nl/en/v1.29.0/"
```

Why not:
```suggestion
        "link": "https://docs.openinwoner.nl/en/latest/"
```
Then we don't have to update the version for every release.

---

Because if users click the link, we want them to arrive in the correct docs for their currently active version. The RTD interface is subtle and it might not be immediately obvious to the median user that the docs are versioned.